app/_src/policies/meshtrafficpermission.md

```diff
@@ -38,12 +38,12 @@ It allows you to define granular rules about which services can communicate with
 | `from[].targetRef.kind` | `Mesh`, `MeshSubset`, `MeshServiceSubset` |
 {% endif_version %}
 {% endtab %}
-{% tab targetRef27x Builtin Gateway %}
+{% tab targetRef Builtin Gateway %}
 `MeshTrafficPermission` isn't supported on builtin gateways. If applied via
 `spec.targetRef.kind: MeshService`, it has no effect.
 {% endtab %}
 
-{% tab targetRef27x Delegated Gateway %}
+{% tab targetRef Delegated Gateway %}
```

Comment:
After https://github.com/kumahq/kuma-website/pull/2216 there shouldn't be the "id" of the tabs anymore, just the tab name:
```suggestion
{% tab Delegated Gateway %}
```